plugins/modules/azure_rm_eventhub_info.py

```diff
@@ -0,0 +1,241 @@
+#!/usr/bin/python
+#
+# Copyright (c) 2021 Praveen Ghuge(@praveenghuge) Karl Dasan(@karldas30) Saurabh Malpani (@saurabh3796)
+#
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+DOCUMENTATION = '''
+---
+module: azure_eventhub_info
+version_added: "1.6.0"
+short_description: Get Azure Event Hub
+description:
+    - Get facts of Azure Event Hub.
+options:
+    resource_group:
+        description:
+            - The name of the resource group.
+        required: True
+        type: str
+    namespace_name:
+        description:
+            - The name of the namspace.
+        required: True
```

Comment:
Same with argument_spec
```suggestion
        required: False
```